src/app/math/page.tsx

```
@@ -0,0 +1,113 @@
+import MathQuestion from "@/components/features/Questions/Question-UI/QuestionModules/MathQuestion";
+import Header from "@/components/features/Questions/Question-UI/Header";
+import StreakAnnouncer from "@/components/features/Questions/Modals/StreakAnnouncer";
+import useQuestionHandler from "@/hooks/questions";
+import Spinner from "@/components/common/Spinner";
+import GetStarted from "@/components/features/Questions/Question-UI/GetStarted";
+import Result from "@/components/features/Questions/Question-UI/Results";
+import { useQuestionStore, useTopicStore } from "@/store/questions";
+import QuestionWrappers from "@/components/wrappers/Question";
+import MainWrappers from "@/components/wrappers/Main";
+
+const Math = () => {
```

Biome (1.9.4)

[error] Do not shadow the global "Math" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)